script.js

```diff
@@ -38,8 +38,8 @@ const addItemsFeatures = (newItem) => {
   newItem.append(itemName, quantity, comments, check, remove);
 };
 
-userInput.addEventListener("keydown", (key) => {
-  if (key.keyCode === 13) {
+userInput.addEventListener("keydown", (event) => {
```

Comment:
all looks good!

BTW if you want to be more elegant with arrow functions , you can remove the parenthesis when you have only one parameter.
```
const addOne = (num) => num + 1
``` 
can be written as:
```
const addOne = n => n +1
```